make/tasks/workspace/lint.sh

```diff
@@ -5,5 +5,5 @@ set -euo pipefail
 SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 ROOT_DIR=$($SCRIPT_DIR/../../utilities/root_dir.sh)
 
-swift run --package-path $ROOT_DIR/projects/tuist/vendor swiftformat $ROOT_DIR --lint
-swift run --package-path $ROOT_DIR/projects/tuist/vendor swiftlint lint --quiet --config $ROOT_DIR/.swiftlint.yml $ROOT_DIR/Sources
+swift run --package-path $ROOT_DIR/projects/tuist/vendor swiftformat $ROOT_DIR
+swift run --package-path $ROOT_DIR/projects/tuist/vendor swiftlint lint --fix --quiet --config $ROOT_DIR/.swiftlint.yml $ROOT_DIR/Sources
```

Comment:
I'd not mix linting and fixing. Fixing doesn't error when there are non-fixable linting issues.

Reply:
Oh really? I didn't expect that, I'll try to check if there is any flag to both fix and report errors 😓